src/style.css

```diff
@@ -0,0 +1,112 @@
+@font-face {
+  font-family: "Avenir-Book";
+  font-weight: 300;
+  color: #3d4e61;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  font-weight: 300;
+  color: #3d4e61;
+  src: url("./fonts/Avenir-Heavy.ttf");
+
+}
+
+.form_field {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  position: relative;
+}
+
+.icon_big {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.field_big {
+  box-sizing: border-box;
+  font-size: 16px;
+  line-height: 22px;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  font-weight: 300;
+  font-family: Avenir-Book, sans-serif;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+}
+
+.field_big::placeholder {
+  color: #3d4e61;
+  font-family: Avenir-Book, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+}
+
+.field_big:hover {
+  background: #ffff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.field_big:focus {
+  background: linear-gradient(180deg, #ffff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.field_small {
+  box-sizing: border-box;
+  font-size: 14px;
+  line-height: 19px;
+  width: 100%;
+  height: 42px;
+  padding-left: 33px;
+  font-weight: 300px;
+  font-family: Avenir-Book, sans-serif;
+  background: #ffff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  outline: none;
+
+}
+
+.icon_small {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.field_small::placeholder {
+  color: #3d4e61;
+  font-family: Avenir-Book, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+
+}
```

Comment:
too many duplications in code, add some common classes and move repeated styles there